homeassistant/components/modbus/validators.py

```diff
@@ -48,7 +50,7 @@
     "ENTRY",
     [
         "struct_id",
-        "register_count",
```

Comment:
Please keep this as register_count, because it should be independent of register_size.

Reply:
The purpose is that register count is now a calculated value for the given register size.

Reply:
No the purpose of register_count is to tell how many registers are needed when requesting data.